shared/chat/conversation/container.js

```diff
@@ -42,7 +55,12 @@ class Conversation extends React.PureComponent<SwitchProps> {
         // To solve this we render a blank screen on mobile conversation views with "noConvo"
         return isMobile ? null : <NoConversation />
       case 'normal':
-        return <Normal conversationIDKey={this.props.conversationIDKey} isPending={this.props.isPending} />
+        return (
+          <>
+            <NavigationEvents onDidFocus={this._onDidFocus} onWillBlur={this._onWillBlur} />
```

Comment:
:sheep: :sheep: At some point would be nice if we could type the context that's required by this component, because I think this will runtime error if navigation doesn't exist in the context. Or, alternately, we can explicitly pass navigation in.